src/pages/ProfilePage/PhotoGallery/Downvote.js

```diff
@@ -0,0 +1,25 @@
+import { Button } from 'primereact/button';
+import PhotoLikesService from '../../../services/photolikesService';
+
+const DownvoteButton = ({ photo, userId, setData, disabled }) => {
+  return (
```

Comment:
I can dislike an image after I liked it. Can you please check it out? The dislike button is disabled when I like an image

![Screen Shot 2023-09-15 at 10 05 37](https://github.com/tonkec/duga_frontend/assets/5020758/eb81ddc5-2903-438e-a367-e8c8ef5cbfda)